README.md

```diff
@@ -7,7 +7,7 @@ You're working on your Tunr app and you encounter this error. What does it mean
 ![Rails error](http://i.imgur.com/9NR7XNT.png)  
 
 ```text
-Your answer...
+It looks like index html pages have not been created for the artist class and the application.  The controller has a "find" function that does not seem to link up to the view.  Therefore, I would go to views/artists and create an index.html.erb file and views/layout and create an index.html.erb file.
 ```
```

Comment:
:+1: although the error only calls for `artists/index.html`, not the additional layout index file.